index.html

```diff
@@ -50,18 +37,202 @@ <h1>I'm Salwa <br>Glad to see you!</h1>
                 me.
             </p>
         </div>
-        <div class="connect">
-            <a href="#"><p class="connect-link">Let's connect</p></a>
+        <div class="connect1">
+            <div class="text1">
+                <a href="#">LET'S CONNECT </a>
+          </div>
+            <div><ul class="social-icons1">
+            		<li><a href="#"><img src="./style/img/github.png" alt="github"></a></li>
+                    <li><a href="#"><img src="./style/img/linkedin .png" alt="linkedin"></a></li>
+                    <li><a href="#"><img src="./style/img/v.png" alt="v sign"></a></li>
+                    <li><a href="#"><img src="./style/img/twitter.png" alt="twitter"></a></li>
+                    <li><a href="#"><img src="./style/img/m.png" alt="m sign"></a></ul>
+            </div>
+    </section>
+</body>                                                                                                                                                                                                                                          
+</html>
+<!--Project illustration-->
+<section class="allcards">
+    <div class="card1">
```

Comment:
Please use a descriptive tag like `<section></section>` or `<article></article>` instead of a div tag. Because using HTML5 descriptive tags you can identify your sections easily.